### PR TITLE
Course CSV Download and Correct option(s) in view answerpaper

### PR DESCRIPTION
**Pull request by maheshgudi**
screenshot from 2017-01-18 15 17 26
1. Moderator can now download the csv dump of the performance of students in each quiz (best attempt of the quiz) for a course.

2. Students, Moderators(in grade user and monitor) can view the correct option(s) for MCQ/MCC questions.

### CODE REVIEW

File: yaksh/templates/yaksh/grade_user.html [hunk 1]
```diff
@@ -130,7 +130,14 @@
             {% if question.type == "mcq" or question.type == "mcc" %}
                 <h5> <u>Choices:</u></h5>
                 {% for testcase in question.get_test_cases %}
-                    <br/><strong>{{ forloop.counter }}. {{ testcase.options|safe }}</strong>
+                    {% if testcase.correct %}
+                    <br/><strong><mark style="background-color:#4cff00">
```
Comment (prabhuramachandran) [inline yaksh/templates/yaksh/grade_user.html]: Please put this in a proper class in the CSS later.

File: yaksh/templates/yaksh/user_data.html [hunk 2]
```diff
@@ -66,7 +66,15 @@
             {% if question.type == "mcq" or question.type == "mcc" %}
                 <h5> <u>Choices:</u></h5>
                 {% for testcase in question.get_test_cases %}
-                    <br/><strong>{{ forloop.counter }}. {{ testcase.options|safe }}</strong>
+                    {% if testcase.correct %}
+                        <br/><strong><mark style="background-color:#4cff00">
```
Comment (prabhuramachandran) [inline yaksh/templates/yaksh/user_data.html]: Same here, put this in a style.

File: yaksh/templates/yaksh/view_answerpaper.html [hunk 3]
```diff
@@ -40,11 +40,24 @@
...
             {% if question.type == "mcq" or question.type == "mcc" %}
-            <h5> <u>Choices:</u></h5>
-            {% for testcase in question.get_test_cases %}
-            <br/><strong>{{ forloop.counter }}. {{ testcase.options|safe }}</strong>
-            {% endfor %}
-            {%endif%}
+                <h5> <u>Choices:</u></h5>
+                {% for testcase in question.get_test_cases %}
+                {% if testcase.correct %}
+                <br/><strong><mark style="background-color:#4cff00">
```
Comment (prabhuramachandran) [inline yaksh/templates/yaksh/view_answerpaper.html]: CSS.

File: yaksh/templates/yaksh/grade_user.html (hunk 1 above)
Comment (maheshgudi) [inline yaksh/templates/yaksh/grade_user.html]: resolved

File: yaksh/templates/yaksh/user_data.html (hunk 2 above)
Comment (maheshgudi) [inline yaksh/templates/yaksh/user_data.html]: resolved

File: yaksh/templates/yaksh/view_answerpaper.html (hunk 3 above)
Comment (maheshgudi) [inline yaksh/templates/yaksh/view_answerpaper.html]: resolved

**Comment by maheshgudi** [comment]
Resolved all grievances. Please consider merging.

**Pull request merged**